src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Button = (props) => {
+    return (
+         <button
+         type= {props.type}
+         onClick={props.handleClick}
+         disabled={props.disabled}
+         className={props.className}>
+             {props.text}
+         </button>
+
+    );
+};
```

Comment:
Dit is nu meer een doorgeefluik. Ten opzichte van het html button element zelf gebeurd er verder niets in dit component. Je had dus net zo goed direct kunnen gebruiken